ENIGMAsystem/SHELL/Widget_Systems/Cocoa/dialogs.cpp

```diff
@@ -64,9 +64,9 @@ static inline string remove_trailing_zeros(double numb) {
 
 void show_error(string errortext, const bool fatal) {
   #ifdef DEBUG_MODE
-  errortext = enigma::debug_scope::GetErrors() + "\n\n" + errortext;
+  errortext = enigma::debug_scope::GetErrors() + "\r\n\r\n" + errortext;
```

Comment:
OSX doesn't care about `\r`, if I understand correctly.

Reply:
Yeah, I knew that already. It must have slipped my mind when I made that change.

Reply:
I believe the same is also true for Linux, iirc.